docs/source/chainerx/install/index.rst

```diff
@@ -21,7 +21,12 @@ Environment variable        Description
 ``CHAINER_BUILD_CHAINERX``  ``1`` to build the ``chainerx`` package along with ``chainer``. ``0`` to skip. Default is ``0``.
 ``CHAINERX_BUILD_CUDA``     ``1`` to build ``chainerx`` with CUDA support. ``0`` to skip. Default is ``0``.
                             See also :ref:`CUDA support <chainerx-install-cuda-support>` section below.
-``CHAINERX_ENABLE_BLAS``    ``1`` to make BLAS enabled. ``0`` to disabled. Default is ``1``.
+``CHAINERX_ENABLE_BLAS``    ``1`` to enable BLAS, ``0`` to disable it. Default is ``1``.
+                            If BLAS is enabled, it is searched for and used if found. If not found, ChainerX will behave as if BLAS
+                            was disabled and use different native implementations.
+``CHAINERX_ENABLE_LAPACK``  ``1`` to enable LAPACK, ``0`` to disable it. Default is ``1``.
+                            If LAPACK is enabled, it is searched for and used if found. If not found, ChainerX will behave as if
+                            LAPACK was disabled and cause runtime errors.
```

Comment:
```suggestion
                            LAPACK was disabled and may cause runtime errors.
```

Reply:
I discussed this PR with @niboshi to be clear about reflecting the actual behavior for now (it always cause runtime errors). As for this case, I'd be fine applying your suggestion but would like to check with him first. What do you say? @niboshi

Reply:
I think @Crissman's version is better (actually I was going to suggest this exact change).

Reply:
Got it, applied!